test/fixtures/rawXml.tests.ts

```diff
@@ -46,4 +46,75 @@ describe('raw xml fixture', () => {
 
         // writeTempFile('raw xml - output.docx', doc);
     });
+
+    it("inserts raw xml content from an array of strings", async () => {
```

Comment:
Perhaps "inserts raw xml content from an array - single string" here and "inserts raw xml content from an array - multiple strings" below?